test/unit/uk/gov/hmrc/exports/services/mapping/goodsshipment/ConsignmentConsignorBuilderSpec.scala

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.services.mapping.goodsshipment
+
+import org.mockito.Mockito.when
+import org.scalatest.{Matchers, WordSpec}
+import org.scalatestplus.mockito.MockitoSugar
+import testdata.ExportsDeclarationBuilder
+import uk.gov.hmrc.exports.models.DeclarationType.DeclarationType
+import uk.gov.hmrc.exports.models.declaration.{Address, ConsignorDetails, EntityDetails}
+import uk.gov.hmrc.exports.models.{Country, DeclarationType}
+import uk.gov.hmrc.exports.services.CountriesService
+import wco.datamodel.wco.dec_dms._2.Declaration
+
+class ConsignmentConsignorBuilderSpec extends WordSpec with MockitoSugar with Matchers with ExportsDeclarationBuilder {
+
+  val mockCountriesService = mock[CountriesService]
+  when(mockCountriesService.allCountries)
+    .thenReturn(List(Country("United Kingdom", "GB"), Country("Poland", "PL")))
+
+  "ConsignorBuilder" should {
+    for (declarationType: DeclarationType <- Seq(DeclarationType.CLEARANCE)) {
```

Comment:
Probably `Seq(DeclarationType.CLEARANCE).map` will be more consistent with the approach that we're using everywhere, but `for` is of course good :)

Reply:
Consistency is good, let me address that

Reply:
Done. Please review again.